tests/pytests/functional/modules/test_cmdmod.py

```diff
@@ -0,0 +1,104 @@
+import base64
```

Comment:
Let's rename this to `tests/pytests/functional/modules/cmdmod/test_powershell.py `